downstream/modules/platform/proc-controller-set-up-azure.adoc

```diff
@@ -1,35 +1,34 @@
 [id="controller-set-up-azure"]
 
-= {Azure} active directory authentication
+= Configuring {Azure} active directory authentication
 
-To set up enterprise authentication for {Azure} Active Directory (AD), you need to obtain an OAuth2 key and secret by registering your organization-owned application from Azure at:
-https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app.
+To set up enterprise authentication for {Azure} Active Directory (AD), you need to obtain an OAuth2 key and secret by registering your organization-owned application from Azure using the link:https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app[Quickstart: Register an application with the Microsoft identity platform]. 
 
-Each key and secret must belong to a unique application and cannot be shared or reused between different authentication backends.
-To register the application, you must supply it with your webpage URL, which is the Callback URL shown in the *Authentication* tab of the *Settings* screen.
+Each key and secret must belong to a unique application and cannot be shared or reused between different authentication backends. To register the application, you must supply it with your webpage URL, which is the Callback URL shown in the Authenticator details for your authenticator configuration. See dxref:gw-display-auth-details[Displaying authenticator details] for instructions on accessing this information.  
 
 .Procedure
 
-. From the navigation panel, select {MenuAEAdminSettings}.
-. Select *Azure AD settings* from the list of *Authentication* options.
+. From the navigation panel, select {MenuAMAuthentication}.
```

Comment:
See global suggestions for this step in examples above. (includes Click **Create Authentication**.)

Reply:
Fixed.